### ISSUE TITLE
fs.move does not support mixed case filenames on windows

### ISSUE THREAD

**Issue by simonm3** [PyFilesystem/pyfilesystem2#344]
Windows files are not case sensitive but the filenames do store capital letters.  Hence when trying to make a filename lower case "move" won't work because the new file already exists. Workaround is to move it twice but maybe that should be part of the fs.move or there should be an fs.rename.

**Comment by lurch** [comment]
If I've understood correctly, are you trying to rename e.g. `MixedCaseFilename` to `mixedcasefilename` ?

**Comment by simonm3** [comment]
yes. it is a windows oddity that they are the same but different.

**Comment by rkhwaja** [comment]
MacOS's APFS is also case-preserving but case-insensitive through finder or terminal. It's not that odd.

**Comment by willmcgugan** [comment]
I think we're going to have to support this case. *sigh*
